tests/integration/pyramid_tests.py

```diff
@@ -139,6 +144,7 @@ def test_trace_headers(self):
             "/example",
             headers={
                 "X-Trace": "1234",
+                "X-Edge-Request": base64.b64encode(SERIALIZED_EDGECONTEXT_WITH_NO_AUTH),
```

Comment:
This should also be `base64.b64encode(SERIALIZED_EDGECONTEXT_WITH_NO_AUTH).decode()` so it's the type that the code expects.